vm-setup/roles/common/defaults/main.yml

```diff
@@ -114,3 +114,6 @@ networks_nated: "{{ provisioning_network_nat + external_network }}"
 networks_nat_only: "{{ lookup('env', 'ENABLE_NATED_PROVISIONING_NETWORK') | bool }}"
 # Placeholder for the eventually selected network composition
 networks: "{{ networks_nated if networks_nat_only else networks_mixed }}"
+
+# Create a SSL certificate and configure sushy to use it
+sushy_use_ssl: "False"
```

Comment:
Is there any reason not to use TLS by default?

(btw, I'd rather call it "tls" than "ssl")

Reply:
Most real-world BMC's use TLS, and in most cases we need to disable the validation, unfortunately. I'd just switch the whole thing to TLS by default (bonus: no need to figure out the right True/true values)